media/css/m24/components/navigation-refresh.scss

```diff
@@ -497,26 +501,42 @@
         }
     }
 
+    .c-menu-button-close {
+        @media #{$mq-md} {
+            right: 15px;
+            left: auto;
+            display: block;
+            top: 15px;
+            @include bidi(((right, 15px, unset), (left, unset, 15px)));
```

Comment:
**suggestion:** `unset` might not be supported by some older browsers, suggest defaulting back to `auto` here for broader support instead?
```suggestion
            @include bidi(((right, 15px, left, auto), (left, auto, right, 15px)));
```